addons/test_mail/tests/test_message_post.py

```diff
@@ -452,8 +452,8 @@ def test_post_post_w_template_mass_mode(self):
             'partner_to': '%s,%s' % (self.partner_2.id, self.user_admin.partner_id.id),
             'email_to': 'test@example.com',
             'email_cc': self.partner_1.email,
-            # After the HTML sanitizer, it will become "<p>Body for: ${object.name}<a href="">link</a></p>"
-            'body_html': 'Body for: ${object.name}<script>test</script><a href="javascript:alert(1)">link</a>',
+            # After the HTML sanitizer, it will become "<p>Body for: {{ object.name }}<a href="">link</a></p>"
```

Comment:
Probably wrongly parsed comment

Reply:
I changed the comment to be `# After the HTML sanitizer, it will become "<p>Body for: <t t-out="object.name" /><a href="">link</a></p>"`.
But this should only happen if we do not have `sanitize=False` and we have a `sanitize_qweb=False`.